src/platform/site-wide/cta-widget/components/messages/mvi/NotAuthorized.jsx

```diff
@@ -22,7 +22,7 @@ function NotAuthorized({ authenticatedWithSSOe }) {
           Verify Your Identity
         </button>
         <p>
-          <a href="/sign-in-faq/#how-to-verify">
+          <a href="/resources/verifying-your-identity-on-vagov/">
```

Comment:
This is shown inside the call-to-action alert boxes on health care pages if the user isn't verified